cumulusci/tasks/push/tasks.py

```diff
@@ -229,15 +235,31 @@ def _init_task(self):
     def _init_options(self, kwargs):
         super(SchedulePushOrgList, self)._init_options(kwargs)
 
+        neither_file_option = "orgs" not in self.options and "csv" not in self.options
+        both_file_options = "orgs" in self.options and "csv" in self.options
+        if neither_file_option or both_file_options:
+            raise TaskOptionsError(
+                "Please call this task with either the `orgs` or `csv` option."
+            )
         # Set the namespace option to the value from cumulusci.yml if not
         # already set
         if "namespace" not in self.options:
             self.options["namespace"] = self.project_config.project__package__namespace
         if "batch_size" not in self.options:
             self.options["batch_size"] = 200
+        if "csv" not in self.options and "csv_field_name" in self.options:
+            raise TaskOptionsError("Please provide a csv file for this task to run.")
 
     def _get_orgs(self):
-        return self._load_orgs_file(self.options.get("orgs"))
+        if "csv" in self.options:
+            with open(self.options.get("csv")) as csvfile:
```

Comment:
```suggestion
            with open(self.options.get("csv"), newline="", encoding="utf-8") as csvfile:
```

CSV files should always be opened with `newline=""` so that the csv module can handle processing of newlines (otherwise newlines inside quoted fields might not get handled correctly). And it's best to explicitly specify encoding="utf-8" because the default file encoding is different on Windows.